Sources/ResilientDecoding/ResilientPropertyErrorInspection.swift

```diff
@@ -37,7 +37,7 @@ extension Resilient.ProjectedValue where Value: ResilientRawRepresentable {
 
 // MARK: - Sequences
 
-extension Resilient.ProjectedValue where Value: _ResilientSequence {
```

Comment:
Even though we're maintaining the same API, this is a breaking change 😬. Note that we should update our version in our podspec when we're making breaking changes (or we can roll all breaking changes into a single version bump).

Reply:
I think underscored properties are special because they scream "this is an implementation detail"

Reply:
Also, isn't this only an issue when bumping the tag? (it is for SPM, and I care very little about CocoaPods).

Reply:
I'm okay adopting a rule on this repo that "changes involving underscored types do not constitute breaking changes". That works for me.

Re CocoaPods: you'll need to update the version in the Podspec _before_ tagging. We should keep these versions in sync. I try to have PRs update the Podspec version to the expected new version. This can be done post-PR, but must be done prior to pushing the git tag.